src/components/buy/payment-information.tsx

```diff
@@ -58,7 +58,10 @@ export function PaymentInformationContent({ info }: PaymentInformationContentPro
           <CopyButton onCopy={() => copy(info.purpose)} />
         </StyledDataTableRow>
       </StyledDataTable>
-      <GiroCode info={info} />
+      <div className="flex flex-col items-center py-4 gap-1">
+        <GiroCode info={info} />
+        <p className="text-dfxBlue-800 font-semibold text-base">GiroCode</p>
+      </div>
```

Comment:
Hm 🤔 now we would show the GiroCode text if the data is not valid, but not the QR code.

Reply:
good point, will move the div into `GiroCode` component...